### ISSUE TITLE
TestSum & TestSub

### ISSUE THREAD

**Issue by khda91** [barton666/calc#10]
Эти тесты проверяют какой-то свой калькулятор?

**Comment by barton666** [comment]
Да, тесты обращались к совершенному другому классу, который использовался на начальном этапе при настройке проекта. 
Исправлено.
